llm-orchestrator-server/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3
+
+COPY target/main/python/app /app
+
+WORKDIR /app
+
+RUN pip install poetry
```

Comment:
Pin the peotry version as mentioned here : https://medium.com/@albertazzir/blazing-fast-python-docker-builds-with-poetry-a78a66f5aed0